### ISSUE TITLE
Sorting does not appear to work for number types.

### ISSUE THREAD

**Issue by JimBobSquarePants** [umco/umbraco-fluidity#24]
Version 1.0.1

Given the following config, I expected the post code list to sort in ascending order based on the `Code` property which is an `int`.  (Post codes in Oz are numeric)

``` c#
treeConfig.AddCollection<PostCode>(
    p => p.Id,
    "PostCode",
    "PostCodes",
    "A collection of post codes", "icon-message-unopened", "icon-message-unopened",
    collectionConfig =>
    {
        collectionConfig.SetNameFormat(p => p.Code.ToString());
        collectionConfig.SetSortProperty(p => p.Code, SortDirection.Ascending); // Explicit ascending
        collectionConfig.SetViewMode(FluidityViewMode.List);

        collectionConfig.ListView(listViewConfig =>
        {
            listViewConfig.AddField(p => p.BandingName).SetHeading(nameof(Banding));
            listViewConfig.AddField(p => p.StateName).SetHeading(nameof(State));
        });

        collectionConfig.Editor(editorConfig =>
        {
            editorConfig.AddTab("General", tabConfig =>
            {
                tabConfig.AddField(p => p.Code).MakeRequired();
                tabConfig.AddField(p => p.StateId).SetDataType("StatePicker").MakeRequired();
                tabConfig.AddField(p => p.BandingId).SetDataType("BandingPicker").MakeRequired();
            });
        });
    });
```

However, this does not create the expected request. Note `orderDirection=desc` in the request url.

```
http://localhost:8000/umbraco/backoffice/fluidity/FluidityApi/getentities?collectionAlias=postCode&orderBy=name&orderDirection=desc&pageNumber=1&pageSize=20&query=&section=database
```

**Comment by mattbrailsford** [comment]
When you say list, are we talking about the listview? or some place else?

**Comment by mattbrailsford** [comment]
Fixed by commit 156294c

**Comment by JimBobSquarePants** [comment]
Yeah listview, sorry I wasn't clear. Thanks for the fix!
